docs/api_reference/service_context/node_parser.rst

```diff
@@ -2,5 +2,9 @@ Node Parser
 ===========
 
 .. automodule:: llama_index.node_parser
+   :members:
+   :inherited-members:
+
+.. automodule:: llama_index.node_parser.extractors
```

Comment:
Can we use pydantic automodule here? It avoids picking up all the messy pydantic functions

example from the LLMs

```
HuggingFaceLLM
==============

.. autopydantic_model:: llama_index.llms.huggingface.HuggingFaceLLM
```

Reply:
yep! thanks